### PR TITLE
Update CMS template ID

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into cqm-reports require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/qrda-export/catI-r5/qrda1_r5.rb [hunk 1]
```diff
@@ -21,14 +21,19 @@ def initialize(patient, measures, options = {})
     @performance_period_end = options[:end_time]
     @submission_program = options[:submission_program]
     @ry2026_submission = options[:ry2026_submission]
+    @ry2027_submission = options[:ry2027_submission]
     @medicare_beneficiary_identifier = options[:medicare_beneficiary_identifier]
     @hicn = options[:hicn]
     @authoring_system = options[:authoring_system]
   end
 
   # use 2026 template ids if 2026 submission is specified as true, or if it unspecified
```
Comment (elsaperelli) [inline lib/qrda-export/catI-r5/qrda1_r5.rb]: Is this still the case? Or can we update this comment?

**Review by elsaperelli — CHANGES_REQUESTED** [review]
One small comment!

File: lib/qrda-export/catI-r5/qrda1_r5.rb (hunk 1 above)
Comment (dczulada) [reply lib/qrda-export/catI-r5/qrda1_r5.rb]: good catch.  comment updated

**Review by elsaperelli — APPROVED** [review]
